### DISCUSSION TITLE
重启代理||清理反代缓存 提示失败  接口显示 502

### DISCUSSION THREAD

**Discussion by wuye251** [1Panel-dev/1Panel#5067] [Q&A]
### 1Panel 版本

1.10.5-lts

### 问题描述

在我添加了几个域名代理后,我重启||清理反代缓存失败(之前有成功过)，一直提示502(1panel后台能登入，代理用的openresety)
![1713796742448](https://github.com/1Panel-dev/1Panel/assets/24435699/4361278f-4db0-4dea-b1b6-f46d59626f39)
![1713796775449](https://github.com/1Panel-dev/1Panel/assets/24435699/2541ca79-869b-4164-b341-b5690b2a3f76)

### 重现步骤

暂无

**Comment by zhengkunwang223** [comment]
看一下运行中的 OpenResty 状态 和 使用的镜像版本
